.github/workflows/publish.yaml

```diff
@@ -142,8 +151,9 @@ jobs:
           export PLATFORM_TAG=$(echo ${{ matrix.docker_platform }} | tr '/' '-')
           docker buildx build \
             --platform=${{ matrix.docker_platform }} \
-            --tag $IMAGE_NAME:${RELEASE_VERSION}  \
-            --tag $IMAGE_NAME:latest \
+            --tag $IMAGE_NAME:${RELEASE_VERSION}-${{ matrix.tag_suffix }}  \
+            --tag $IMAGE_NAME:latest-${{ matrix.tag_suffix }} \
```

Comment:
We kind of need the image name to be latest still, otherwise down stream will not be able to pull latest

Were you able to execute this locally?